### ISSUE TITLE
[vkmk] Update workgroups at login from vokomokum

### ISSUE THREAD

**Issue by wvengen** [foodcoop-adam/foodsoft#154]
When a member logs into Foodsoft using the vokomokum login, the user's groups are available from the userinfo endpoint. On each login, Foodsoft needs to update the workgroups a member is part of based on that info.

**Comment by wvengen** [comment]
Member's [userinfo endpoint](http://members.vokomokum.nl/userinfo) returns group info, like:

```
{
(...)
"groups": [
    {
        "coordinator": true,
        "name": "Systems",
        "id": 1
    }
]
}
```

Let's match groups by name. If a group doesn't exist in Foodsoft, it's as if the user is not part of that group. Foodsoft has no such thing as a group coordinator, we need to check if that is needed or not.

**Comment by wvengen** [comment]
Implemented in 3687beb79642c7516cc7c13686aceb212227dfed.
